docs/security/how-to-use-claimmapper.md

```diff
@@ -0,0 +1,103 @@
+---
+id: how-to-use-claimmapper
+title: How to Use ClaimMapper
+sidebar_label: How to Use ClaimMapper
+---
+
+`ClaimMapper` is a plugin that extracts claims from JSON Web Tokens (JWT). This process is achieved with the method `GetClaims`, which translates `AuthInfo` structs from the caller into `Claims` about the caller's roles within Temporal.
+
+A `Role` (within Temporal) is a bit mask that combines one or more of the role constants.
+
+```go
+role := authorization.RoleReader | authorization.RoleWriter
+```
+
+`GetClaims` is customizable, and can be modified with the `temporal.WithClaimMapper` server option. Temporal also offers a default JWT `ClaimMapper` for your use.
+
+A typical approach is for `ClaimMapper` to interpret custom `Claims` from a caller's JWT access token, such as membership in groups, and map them to Temporal roles for the user. The subject information from the caller's TLS certificate can also be a parameter in determining roles.
```

Comment:
> caller's JWT access token,

Their JSON Web Token access token? 😉 

I think this is the first time JWT is used in this section, so maybe `JSON Web Token (JWT)` with a link out works and actually, as I write this, I spot a usage of JWT on the line before.

So that JWT should become `JSON Web Token (JWT)`, and this one whole comment can be shortened down to ~~access token~~. 🤷‍♂️

Reply:
Gotcha.